.github/workflows/flakehub-publish-tagged.yml

```diff
@@ -0,0 +1,17 @@
+on:
+  workflow_dispatch:
+  push:
```

Comment:
Can we make this only run for branches named `release-branch/*`?

At some point I expect to start tagging `main` with unstable builds 1.49.X, because tracking what is in each unstable has been a pain point. We wouldn't want to accidentally push those to Nix.

Reply:
Unfortunately, the semantics require a tag. What if we matched against even minor versions?

```
'^v\d+\.\d*[02468]+\.\d+$'
```

Reply:
Triggering only on even numbered branches would solve the issue, if that is the only way to make it work.

This doesn't work?
```
on:
  push:
    branches:
      - "release-branch/*"
    tags:
      - "v*.*.*"
```

Reply:
I believe this will run either branches OR tags.

Reply:
Changed it to `^v[0-9]+\.[0-9]*[02468]+\.[0-9]+$` because `\d` might be too cute for GitHub